server/sonar-web/src/main/js/apps/update-center/init.js

```diff
@@ -31,8 +31,9 @@ import Plugins from './plugins';
 const App = new Marionette.Application();
 const init = function (el) {
   // State
+  // FIXME window.SS.updateCenterActive
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1134)